tests/integration/tier0/check-user-response/test_user_response.py

```diff
@@ -1,8 +1,12 @@
+import platform
+
 from envparse import env
 
 
-def test_check_user_response_user_and_password(convert2rhel):
+booted_os = platform.platform()
```

Comment:
The `booted_os` variable is used in this test? If not then remove it please (the import of `platform` as well) :D

Reply:
done!